lib/ansible/plugins/inventory/yc_compute.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: yc_compute
+    plugin_type: inventory
+    short_description: Yandex.Cloud Compute inventory source
+    requirements:
+      - yandexcloud==0.10.1
+    extends_documentation_fragment:
+      - constructed
+      - inventory_cache
+    description:
+      - Pull inventory from Yandex Cloud Compute.
+      - Uses a YAML configuration file that ends with yc_compute.(yml|yaml) or yc.(yml|yaml).
+    version_added: "2.10"
+    options:
+      auth_kind:
+        description: The type of credential used.
+        required: True
+        choices: ['oauth', 'serviceaccountfile']
+        env:
+          - name: YC_ANSIBLE_AUTH_KIND
+      oauth_token:
+        description: OAUTH token string. See U(https://cloud.yandex.com/docs/iam/concepts/authorization/oauth-token).
+        type: string
+        no_log: True
```

Comment:
There's no harm in having this here, but thought I'd mention that I don't think `no_log: True` does anything in documentation. As long as you aren't outputting the option in this plugin it shouldn't be exposed. Tangentially, I have a PR to add support for inline vaulted options for inventory plugins here #60715.

Reply:
Ok, removed no_log entries from doc by now.